Cue/app/tabs/library/play/PlayDeckSetupView.js

```diff
@@ -37,23 +39,88 @@ export default class PlayDeckSetupView extends React.Component {
 
   state: {
     playbackOption: PlaybackOption,
-    customStartIndex: number
-  }
-
-  _onPlaybackOptionSelected = (option: PlaybackOption) => {
-    this.setState({
-      ...this.state,
-      playbackOption: option
-    })
+    customStartIndex: number,
+    flaggedOnly: boolean,
+    answerFirst: boolean,
+    filteredCards: Array<Card>,
   }
 
   constructor(props: Props) {
     super(props)
 
-    this.state = {
+    let state = {
       playbackOption: 'sequential',
-      customStartIndex: 0
+      flaggedOnly: this.props.flagFilter,
+      answerFirst: false,
+      customStartIndex: 0,
+    }
+    state.filteredCards = this._generateDeckFilter(state)(this.props.deck.cards || [])
```

Comment:
Sort of dumb, but if you want this to type-check with Flow, you'll need to pre-create the `filteredCards` key in the object literal and assign it a dummy `[]` value before immediately overwriting it with the filtered cards.